documentation/slate/source/includes/disruptions.md

```diff
@@ -0,0 +1,62 @@
+Disruptions
+===========
+
+
+In this paragraph, we will know how the disruptions are displayed in the different APIs
+
+# Context
+
+## periods
+
+### production period 
+The production period is global to a navitia coverage and is the validity period of the coverage's data.
+This production period cannot excede one year.
+
+### publication period
+The publication period of a disruption is the period on which we want to display the disruption in navitia.
+
+The creator of the disruption might not want the traveller to know of a disruption before a certain date (because it's too uncertain, secret, ...)
+The publication period is the way to control this
+
+### application periods
+The application periods are the list of periods on which the disruption is active
```

Comment:
missing final `.`
We may add that `For example it's the actual period when railworks are done and train circulation is cut.`